<ui version="4.0" >
 <class>ShaderDialog</class>
 <widget class="QDialog" name="ShaderDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>255</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="MinimumExpanding" hsizetype="Preferred" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>GLSL Shaders</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <widget class="QTabWidget" name="tabWidget" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="loadTab" >
      <attribute name="title" >
       <string>Load Shaders</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout" >
       <item row="0" column="0" >
        <widget class="QLabel" name="label_5" >
         <property name="text" >
          <string>Shader Name:</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="1" >
        <layout class="QHBoxLayout" name="horizontalLayout_5" >
         <item>
          <widget class="QLineEdit" name="shaderName" />
         </item>
        </layout>
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="label_4" >
         <property name="text" >
          <string>Vertex Shader:</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <layout class="QHBoxLayout" name="horizontalLayout_4" >
         <item>
          <widget class="QLineEdit" name="vertFile" />
         </item>
         <item>
          <widget class="QPushButton" name="vertFileButton" >
           <property name="text" >
            <string>Browse...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0" >
        <widget class="QLabel" name="label_7" >
         <property name="text" >
          <string>Fragment Shader:</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="1" >
        <layout class="QHBoxLayout" name="horizontalLayout_7" >
         <item>
          <widget class="QLineEdit" name="fragFile" />
         </item>
         <item>
          <widget class="QPushButton" name="fragFileButton" >
           <property name="text" >
            <string>Browse...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="0" >
        <widget class="QLabel" name="label_8" >
         <property name="text" >
          <string>Parameters:</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="1" >
        <layout class="QHBoxLayout" name="horizontalLayout_9" >
         <item>
          <widget class="QLineEdit" name="paramsFile" />
         </item>
         <item>
          <widget class="QPushButton" name="paramsFileButton" >
           <property name="text" >
            <string>Browse...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="4" column="1" >
        <spacer name="verticalSpacer" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>18</width>
           <height>34</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="5" column="0" colspan="2" >
        <layout class="QHBoxLayout" name="horizontalLayout" >
         <item>
          <spacer name="horizontalSpacer_2" >
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QDialogButtonBox" name="loadVertButton" >
           <property name="standardButtons" >
            <set>QDialogButtonBox::Open</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="shaderTab" >
      <attribute name="title" >
       <string>Assign Shader</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2" >
       <item>
        <layout class="QGridLayout" name="gridLayout_2" >
         <item row="0" column="0" >
          <widget class="QLabel" name="label" >
           <property name="text" >
            <string>Display Type:</string>
           </property>
           <property name="alignment" >
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="1" >
          <widget class="QComboBox" name="displayTypes" >
           <property name="sizeIncrement" >
            <size>
             <width>1</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="1" column="0" >
          <widget class="QLabel" name="label_2" >
           <property name="text" >
            <string>Shader Program:</string>
           </property>
           <property name="alignment" >
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="1" >
          <widget class="QComboBox" name="shaderPrograms" >
           <property name="sizeIncrement" >
            <size>
             <width>1</width>
             <height>0</height>
            </size>
           </property>
           <item>
            <property name="text" >
             <string>None</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>104</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2" >
         <item>
          <spacer name="horizontalSpacer" >
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="shaderButton" >
           <property name="text" >
            <string>Assign Shader</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>shaderName</tabstop>
  <tabstop>vertFile</tabstop>
  <tabstop>vertFileButton</tabstop>
  <tabstop>fragFile</tabstop>
  <tabstop>fragFileButton</tabstop>
  <tabstop>paramsFile</tabstop>
  <tabstop>paramsFileButton</tabstop>
  <tabstop>loadVertButton</tabstop>
  <tabstop>displayTypes</tabstop>
  <tabstop>shaderPrograms</tabstop>
  <tabstop>shaderButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
